src/Models/restaurants.json

```diff
@@ -0,0 +1,141 @@
+[  
+    {
+        "restaurantName": "MBM Trinbago Meals",
+        "address": "Pool Road, Birmingham B66 3DB",
+        "phoneNumber": {
+            "landline": null, 
+            "mobile": "07792557204"
+        },
+        "cusineCategory": "Trinidadian, dine-in and takeaway", 
```

Comment:
Maybe cusineCategory could be arranged in a different way. It seems like a description at the moment, and it could be helpful to have it in an array. Apart from that, I love the structure of the Json file! Great work!

Reply:
Thanks Hawa! That's a great idea, it makes more sense to arrange it in an array like you said, I'll get on and change it. The array could include things like whether you can dine in or not, whether they offer takeaway and type of cuisine.

Reply:
sounds great!